lib/rules/declaration-property-value-disallowed-list/index.js

```diff
@@ -32,20 +32,19 @@ const rule = (primary) => {
 			return;
 		}
 
+		const propKeys = Object.keys(primary);
```

Comment:
[note] Calling `Object.keys(primary)` multiple times in `walkDecls()` loop is wasteful.